products/sle/main.pm

```diff
@@ -469,7 +469,7 @@ sub load_inst_tests() {
             && sle_version_at_least('12-SP2')
             && is_server()
             && (!is_sles4sap() || is_sles4sap_standard())
-            && install_this_version())
+            && (install_this_version() || install_to_other_at_least('12-SP2')))
```

Comment:
Yes, you are right. This is a bug fix, rather than a enhancement to make test moe robust over ipmi. I just combine the two fix together.

I will add comment for it.